### PR TITLE
[Docs] Fix api docs in readthedocs

### PR DESCRIPTION
**Pull request by yl-1993**
- The current version https://mmhuman3d.readthedocs.io/en/latest/api.html cannot display normally
- The revised version is demo on https://mmhuman3dyl-1993.readthedocs.io/en/latest/api.html, which displays APIs normally.

### CODE REVIEW

**Comment by Maoxie** [comment]
Nice revision (๑•̀ㅂ•́)و✧

File: mmhuman3d/utils/__init__.py [hunk 1]
```diff
@@ -1,7 +1,112 @@
...
+    rot6d_to_quat,
+    rot6d_to_rotmat,
+    rotmat_to_aa,
+    rotmat_to_ee,
+    rotmat_to_quat,
+    rotmat_to_rot6d,
+    sja_to_aa,
+)
 
 __all__ = [
```
Comment (Maoxie) [inline mmhuman3d/utils/__init__.py:88]: The `__all__` looks very long and may be prone to cause some difficulties for maintenance?

Comment (yl-1993) [reply mmhuman3d/utils/__init__.py:88]: The `__init__.py` is generated by [mkinit](https://pypi.org/project/mkinit/), which can be used for maintenance.

Comment (Maoxie) [reply mmhuman3d/utils/__init__.py:88]: That solves the problem.

**Review by Maoxie — APPROVED** [review]

**Pull request merged**